platform/shared/access-control.ts

```diff
@@ -263,11 +262,6 @@ export const RouteId = {
   GetAgentStatistics: "getAgentStatistics",
   GetModelStatistics: "getModelStatistics",
   GetOverviewStatistics: "getOverviewStatistics",
-  // Organization Routes
-  GetOrganizationAppearance: "getOrganizationAppearance",
-  UpdateOrganizationAppearance: "updateOrganizationAppearance",
-  UploadOrganizationLogo: "uploadOrganizationLogo",
-  DeleteOrganizationLogo: "deleteOrganizationLogo",
```

Comment:
we don't need 6 routes for `/api/organization`, the 2 routes left in this PR can do all of these operations in the same manner